src/core/app.lua

```diff
@@ -236,6 +236,15 @@ function main (options)
       assert(not done, "You can not have both 'duration' and 'done'")
       done = lib.timer(options.duration * 1e9)
    end
+
+   local breathe = breathe
+   if options.measure_latency or options.measure_latency == nil then
+      local histogram = require('lib.histogram')
```

Comment:
From the conservative perspective:

* The `require` should be at the top of the file (I like to use local requires in selftests but I am not sure if we should expand the use of this pattern)
* `core.*` should not require `lib.*`, @wingo pointed out a precedent in https://github.com/SnabbCo/snabbswitch/pull/764#issuecomment-188227602 but I consider it to be a bug
* thus: `lib.histogram` should depend on `core.shm` and be `core.histogram` (and reuse `core.shm.root`, see `core.counter`)